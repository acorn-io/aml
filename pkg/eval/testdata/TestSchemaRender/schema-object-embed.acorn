schema {
    y?: {
        b: 1
    }
    a: {
        a: "hi"
        y
    }
}

a: b: 2
