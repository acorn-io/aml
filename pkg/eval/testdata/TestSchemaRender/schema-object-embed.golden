{
  "fields": [
    {
      "name": "y",
      "type": {
        "kind": "object",
        "object": {
          "path": "embedded.0.y",
          "fields": [
            {
              "name": "b",
              "type": {
                "kind": "number",
                "default": 1
              }
            }
          ]
        }
      },
      "optional": true
    },
    {
      "name": "a",
      "type": {
        "kind": "object",
        "object": {
          "path": "a",
          "fields": [
            {
              "name": "b",
              "type": {
                "kind": "number",
                "default": 2
              }
            }
          ]
        }
      }
    }
  ]
}